CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+This project is maintained by the contribution guidelines identified for
+  [chef](https://github.com/chef/chef) project. You can find the guidelines here:
+
+https://github.com/chef/chef/blob/master/CONTRIBUTING.md
+
+## Maintainers
+
+- [Thom May](https://github.com/thommay)
+- [Patrick Wright](https://github.com/patrick-wright)
+- [Serdar Sutay](https://github.com/sersut)
+- [Seth Chisamore](https://github.com/schisamo)
```

Comment:
We need to flesh out how many :+1: it takes to proceed with a merge/accept. Take a look at:
https://github.com/chef/omnibus/blob/master/MAINTAINERS.md#maintainers